src/averageAges.js

```diff
@@ -14,12 +20,18 @@
  *
  * @return {number}
  */
+const MALE_SEX = 'm';
+const FEMALE_SEX = 'f';
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mensOfCenturyArray = people
+    .filter(person => person.sex === MALE_SEX)
+    .filter(person => century
+      ? century === Math.ceil(person.died / 100) : true);
```

Comment:
Don't use literals in expressions (HARDCODE).

you can put 100 into a variable